features/steps/jobs_steps.go

```diff
@@ -10,36 +10,42 @@ import (
 	"strings"
 	"time"
 
-	"io/ioutil"
-	"net/http"
-
-	componenttest "github.com/ONSdigital/dp-component-test"
+	componentTest "github.com/ONSdigital/dp-component-test"
 	"github.com/ONSdigital/dp-healthcheck/healthcheck"
 	"github.com/ONSdigital/dp-search-reindex-api/config"
 	"github.com/ONSdigital/dp-search-reindex-api/models"
+	"github.com/ONSdigital/dp-search-reindex-api/mongo"
 	"github.com/ONSdigital/dp-search-reindex-api/service"
-	"github.com/ONSdigital/dp-search-reindex-api/service/mock"
+	serviceMock "github.com/ONSdigital/dp-search-reindex-api/service/mock"
+	"github.com/benweissmann/memongo"
 	"github.com/cucumber/godog"
 	"github.com/pkg/errors"
 	"github.com/rdumont/assistdog"
 	uuid "github.com/satori/go.uuid"
 	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"net/http"
```

Comment:
I'd move these two standard library imports up to the first block of imports (after "time").

Reply:
thanks for pointing this out